reseq_ros2/reseq_ros2/communication.py

```diff
@@ -150,19 +172,29 @@ def topic_from_id(self, id: int) -> rc.ReseQTopic:
     def topic_from_name(self, name: str) -> rc.ReseQTopic:
         return next(filter(lambda x: x.name == name, rc.topics))
 
+    def destroy(self):
```

Comment:
I added this function to solve the `rclpy._rclpy_pybind11.RCLError: failed to shutdown: rcl_shutdown already called on the given context, at ./src/rcl/init.c:241` exception. Now the monitoring function will be shut down without problem.

Reply:
See the comments in the new review